go.mod

```diff
@@ -201,3 +201,5 @@ replace (
 
 // Pinned so it includes fix for Windows Nano
 replace k8s.io/klog => k8s.io/klog v1.0.1-0.20200310124935-4ad0115ba9e4
+
+replace github.com/DataDog/agent-payload => github.com/DataDog/agent-payload v0.0.0-20200722131956-8c38b0f51477
```

Comment:
Note: will bump this before merging to point to the latest agent-payload released including the corresponding definitions.

Reply:
Done